src/main/python/resources/annotaters/manual.py

```diff
@@ -0,0 +1,50 @@
+from src.main.python.core.logger import Logger
+import pandas as pd
+import os
+from src.main.python.resources.utilities.utilities import get_project_root_dir,\
+    load_modules, importer
+
+
+class ManualAnnotator:
+    def __init__(self):
+        self.log = Logger().logger
+        self.vep_annotators = []
+        self.location = os.path.join(get_project_root_dir(),
+                                     'src',
+                                     'main',
+                                     'python',
+                                     'resources',
+                                     'annotaters',
+                                     'vep')
+        self._load_vep_annotators()
+
+    def _load_vep_annotators(self):
+        python_modules = load_modules(self.location)
+        self._check_n_modules(python_modules)
+        loaded_python_modules = importer(python_modules, path=self.location)
+        self._check_n_modules(loaded_python_modules)
+        for module in loaded_python_modules:
+            if 'name' in dir(module) and module.usable:
+                self.vep_annotators.append(module)
+        self._check_n_modules(self.vep_annotators)
```

Comment:
> Add check whether n available modules (excluding templates) == n loaded modules?

Would mean that I have to include a test for the `utilities` function `importer(usable_modules, path)`, which can be done with relative ease. I could do some more with error handling in `manual.py` too, `try: dataset = processor.process(dataset) except Exception as e: self.log.warning(f'{processor.name} raised {e} when trying to process.')`

> `load_modules(path)` also seems to exclude modules ending with `abstract.py`, so perhaps using this could make the `if 'name' in dir(module)` unneeded (not sure if this could cause problems elsewhere though)?

This is true. `if 'name' in dir(module)` is also checked within the `importer(usable_modules, path)` function, so that would mean just a `if module.usable:`